cilium/cmd/config.go

```diff
@@ -46,6 +49,7 @@ var configCmd = &cobra.Command{
 func init() {
 	RootCmd.AddCommand(configCmd)
 	configCmd.Flags().BoolVarP(&listOptions, "list-options", "", false, "List available options")
+	configCmd.Flags().IntVarP(&numPages, "num-pages", "n", 0, "Number of pages for perf ring buffer")
```

Comment:
The "0" default has a special meaning. Document it in the flag's doc string, e.g. "Number of pages [...]; if 0, defaults to the Cilium daemon's configuration"